packages/react/src/components/radio-card-group/radio-card.tsx

```diff
@@ -1,5 +1,5 @@
-import {
-    ChangeEvent,
+import React, {
+    ChangeEvent, ReactElement,
```

Comment:
Encore une fois que ça passe le lint 🤔 
```suggestion
import {
    ChangeEvent, 
    ReactElement,
    ReactNode
    isValidElement
```